lib/hathifile_history/records.rb

```diff
@@ -99,7 +99,7 @@ def add_record(rec)
     # @param [String] line A hathifile_line from a hathifile
     # @return [Array<String, Integer>] The htid and recid in this hathifile_line
     def ids_from_line(line)
-      htid, recid_str = line.chomp.split(/\t/, 4).values_at(0, 3)
+      htid, recid_str = line.chomp.split("\t", 5).values_at(0, 3)
```

Comment:
Can we simply leave out the size parameter to `split`? I don't understand the benefit of it here.